www/docs/customization/ko.md

```diff
@@ -103,6 +103,11 @@ kos:
     # Valid options are: spdx and none.
     sbom: none
 
+    # Path to file where the SBOM will be written
+    #
+    # Default: unset - no SBOM written
```

Comment:
could probably clarify that it'll still create the sbom on the image itself, just not in the filesystem.